wazuh/wazuh-kibana-app
# Fix/3564 fix visualization pinned agent

Hi guys,

With this PR we solve the problem of blank visualizations when we pinned an agent and we solve that the agent persist in the URL when you unpinned it.

To test it:
- Try to pin an agent from a different place of SCA or Office pannel and make sure that visualizations are working properly.
- Try to unpin an agent from Office panel and check if the URL has changed too.
